client/src/components/Patient/PatientManageMedicalRecords.tsx

```diff
@@ -0,0 +1,427 @@
+import React, { FunctionComponent, useContext, useState ,useEffect} from 'react';
+import ReactPaginate from 'react-paginate';
+import { HealthContext } from '../../providers/HealthProvider';
+import { AuthContext } from '../../providers/AuthProvider';
+import AWS from 'aws-sdk';
```

Comment:
why is this needed on the front end?

Reply:
removed aws-sdk import. Also I'm not using react-paginate. So removed that as well.